packages/wonder-blocks-core/src/components/text.tsx

```diff
@@ -1,3 +1,5 @@
+// WARNING: If you modify this file you must update the following file:
+// build-settings/overrides/wonder-blocks-core/components/text.js.flow
```

Comment:
```suggestion
// src/components/text.js.flow
```
Need to update this!